testassets/InteropTestsClient/RunGrpcTests.ps1

```diff
@@ -3,11 +3,15 @@
     [bool]$use_tls = $false,
     [bool]$use_winhttp = $false,
     [bool]$use_http3 = $false,
-    [string]$framework = "net6.0",
+    [bool]$publish_aot = $false,
```

Comment:
Are you planning on adding a PR test run with this set to true?

Reply:
The interop tests are run on google infrastructure. I will organize running them twice, with and without AOT, soon.